site/js/modules/webui/input.js

```diff
@@ -294,6 +298,14 @@ class FieldInstance {
     // Validate the value of the input.
     // Throws an error on invalid input values.
     validate() {
+        // Validate file inputs for testing.
```

Comment:
This is not a sufficient comment for the trickiness here.
I don't think this comment will sufficiently inform a reader of this code.